lib/cookie.ts

```diff
@@ -0,0 +1,9 @@
+import { cookies } from "next/headers"
+
+export function getCookies() {
+  const store = cookies();
+  return {
+    token: store.get("token")?.value,
+    uid: store.get("uid")?.value,
+  }
+}
```

Comment:
Is this function being used? It looks like mostly `cookies().get("token")?.value` is being used.

Should we make this a type and only use helpful functions to access cookies?

That way, if we ever want to do something every time a cookie is accessed (e.g. preprocessing or logging), we could still do it :)

Reply:
@viethung7899 did you address this in another comment?